src/components/ResultsTable.tsx

```diff
@@ -18,27 +18,156 @@ export function ResultsTableWidget (props: ResultsTableProps) {
+                                //  let nameOnTitleProperty = result.NameOnTitlePropertyDetails?.find((owner) => owner.Property.id === result.id)
+                                 let nameOnTitleContact_list = result.NameOnTitleContactDetails;
+                                 console.log(nameOnTitleProperty_list.length);
+                                if (nameOnTitleProperty_list.length > 0 ){
+                                    return (
+                                        <tr key={`${result.id}-${index}`}>
+                                            <td>{index + 1}</td>
+                                            <td>{propertyAddress}</td>
+                                            
+                                            
+                                            <td>
+                                                <table>
+                                                <tr>
+                                                    
+                                                    <th className={'nested_table'}>Name on Title </th>
+                                                   
+                                                    
+                                                    <th className={'nested_table'}>Contacts</th>
+                                                    
+                                                   
+                                                </tr>
+                                                
+                                                {nameOnTitleProperty_list.map((property_details, index_property) => (
+                                                    <tr data-index={index_property}> 
+                                                        <td>{property_details.Name_On_Title.name || 'Name on Title is not found'} &nbsp;
+                                                             <br></br>
+                                                            {/* {JSON.stringify(nameOnTitleProperty_list)} */}
+                                                        </td>  
+                                                        
+                                                        <td>
+                                                        
+                                                        {nameOnTitleContact_list.map((contact_details, index_contact) => {
+                                                            if (nameOnTitleContact_list.length > 0){
+                                                                console.log(JSON.stringify(nameOnTitleContact_list).includes(contact_details.Name_On_Title.id));
+                                                                // let list_str = "".join(str(elem) for elem in nameOnTitleContact_list)
+                                                                if (JSON.stringify(nameOnTitleContact_list).includes(contact_details.Name_On_Title.id) === false){
```

Comment:
more standard to use `!JSON.stringify(nameOnTitleContact_list).includes(contact_details.Name_On_Title.id)`

Reply:
fixed